src/analysis/classic/hunter/shared/KillShot.tsx

```diff
@@ -0,0 +1,77 @@
+import SPELLS from 'common/SPELLS/classic/hunter';
+import { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import ExecuteHelper from 'parser/shared/modules/helpers/ExecuteHelper';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+import { KILL_SHOT_EXECUTE_RANGE } from './constants';
+
+class KillShot extends ExecuteHelper {
+  static executeSources = SELECTED_PLAYER;
+  static lowerThreshold = KILL_SHOT_EXECUTE_RANGE;
+  //static singleExecuteEnablers: Spell[] = [TALENTS.HUNTERS_PREY_TALENT, SPELLS.DEATHBLOW_BUFF];
+  static modifiesDamage = false;
+
+  static dependencies = {
+    ...ExecuteHelper.dependencies,
+    abilities: Abilities,
+  };
+
+  maxCasts: number = 1;
+
+  activeKillShotSpell = SPELLS.KILL_SHOT;
+
+  protected abilities!: Abilities;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = true;
+
+    this.addEventListener(Events.fightend, this.adjustMaxCasts);
+    const ctor = this.constructor as typeof ExecuteHelper;
+    ctor.executeSpells.push(this.activeKillShotSpell);
+
+    (options.abilities as Abilities).add({
+      spell: SPELLS.KILL_SHOT.id,
+      category: SPELL_CATEGORY.ROTATIONAL,
+      charges: 1,
+      // Glyph of Kill Shot reduces by 6 seconds
+      // cooldown: 15,
```

Comment:
Is the `cooldown: 15` comment needed?

Reply:
No. the comment on the Glyph should be enough to explain why the cooldown is not the default for the spell.
Done